src/tests/ftest/util/test_utils_container.py

```diff
@@ -596,6 +596,8 @@ def write_objects(self, rank=None, obj_class=None):
                 self, self.record_qty.value, self.akey_size.value,
                 self.dkey_size.value, self.data_size.value, rank, obj_class,
                 self.data_array_size.value)
+        # Mark the pool as disconnected after API operations
+        self.pool.connected = False
```

Comment:
Talked with @phender and we think a safer change here is
```suggestion
        self.close()
```
And then we still need to test everything using `write_objects`
```
Test-tag: RbldContainerCreate EcodDisabledRebuildSingle PoolEvictTest RbldCascadingFailures RbldContRfTest NvmeObject DmgSystemCleanupTest EcodOfflineRebuildSingle RbldReadArrayTest DaosObjectQuery ContRfEnforce RbldDeleteObjects BoundaryPoolContainerSpace RbldBasic PoolServicesFaultInjection
```

Unfortunately, some of those rebuild tests I think I have known failures so we'll need to ignore those if they fail.